app/components/personal_details_component.rb

```diff
@@ -51,10 +52,19 @@ def email_row
   end
 
   def phone_number_row
-    {
+    basic_row = {
       key: 'Phone number',
       value: phone_number || MISSING,
     }
+
+    if current_namespace == 'support_interface'
```

Comment:
I think we should duplicate this component if we're making them namespace-specific. It's already a bit questionable that we share logic-heavy components between namespaces (see https://github.com/DFE-Digital/apply-for-teacher-training/pull/2071 for some considerations).

We could make the duplication less severe by pushing some stuff to the model.

Reply:
Agreed!